sdk/metric/config.go

```diff
@@ -43,3 +47,13 @@ type resourceOption struct {
 func (o resourceOption) Apply(config *Config) {
 	config.Resource = o.Resource
 }
+
+func WithMetricsProcessors(processors []MetricsProcessor) Option {
```

Comment:
```suggestion
func WithMetricsProcessors(processors ...MetricsProcessor) Option {
```

I would make this signature variadic to simplify the interface for consumers.